roles/repository/defaults/main.yml

```diff
@@ -1 +1,3 @@
 ---
+repository_keys_file: []
+repository_keys_file_dir: "/opt/configuration/environments/generic/repokeys"
```

Comment:
Is this parameter required? It is not used in the ``Add repository keys via file`` task.

Reply:
thanks for the hint, I want to use this, see in task in the new commit